examples/make_generic_contacts.py

```diff
@@ -93,28 +97,71 @@ def make_contacts_by_social_layers_and_age_mixing(n=1e4,location='seattle_metro'
     modeling framework. 
 
     """
-    popdict = sp.make_popdict(n=n,location=location,state_location=state_location,country_location=country_location,use_demography=True)
+    popdict = sp.make_popdict(n=n, location=location, state_location=state_location, country_location=country_location,
+                              use_demography=True)
 
-    n_contacts_dic = {'H': 7, 'S': 20, 'W': 20, 'C': 10} # dict of the average number of contacts per layer
+    n_contacts_dic = {'H': 7, 'S': 20, 'W': 20, 'C': 10}  # dict of the average number of contacts per layer
     options_args = {'use_age_mixing': True, 'use_social_layers': True}
 
-    contacts = sp.make_contacts(popdict,n_contacts_dic=n_contacts_dic,location=location,state_location=state_location,country_location=country_location,sheet_name=sheet_name,options_args=options_args)
-    
+    popdict = sp.make_contacts(popdict, n_contacts_dic=n_contacts_dic, location=location,
+                               state_location=state_location, country_location=country_location, sheet_name=sheet_name,
+                               options_args=options_args)
+
     if verbose:
-        layers = ['H','S','W','C']
-        uids = contacts.keys()
+        layers = ['H', 'S', 'W', 'C']
+        uids = popdict.keys()
         uids = [uid for uid in uids]
         for uid in uids:
             print(uid)
             for k in layers:
-                print(k,contacts[uid]['contacts'][k])
+                print(k, popdict[uid]['contacts'][k])
+
+    return popdict
 
-    return contacts
 
+def describe_population(popdict):
+    ages = []
+    males = 0
+    locations = {}
+    contact_types = {}
+    for peep in popdict:
```

Comment:
I like peep, but maybe stick to uid to match the rest of SynthPops and Covasim?

Reply:
Oh, that makes sense. I was going to go with "person," but I suppose all I'm getting is the key, so uid is a better descriptor.

Reply:
FWIW: It might not be a terrible idea to have some kind of method like this made generally available, an answer to the question "How would you describe this population in a minute or so"

Reply:
I'm making this a tiny bit nicer by adding a check to only list layers that contain contacts, newer output looks like this for totally random:
>Make popdict of random people:
	Ages! mean: 49.88984492246123 min: 0.0 max: 100.0
	Num male: 4965
	locations: {}
	Contacts:

And like this for the complicated one:
> Make contacts by social layers and age mixing
	Ages! mean: 38.39911973592078 min: 0 max: 100
	Num male: 4966
	locations: {}
	Contacts:
		contact type H count: 9942
		degrees: mean: 6.00100583383625 min: 1 max: 24
		contact type W count: 7348
		degrees: mean: 16.363636363636363 min: 1 max: 43
		contact type C count: 9993
		degrees: mean: 10.047233063144201 min: 1 max: 26
		contact type S count: 4521
		degrees: mean: 9.565804025658041 min: 1 max: 44

Reply:
Done (I believe)